tests/apollo/util/concord_msgs.py

```diff
@@ -4,7 +4,7 @@
 
 # Concord
 #
-# Copyright (c) 2020 VMware, Inc. All Rights Reserved.
```

Comment:
Just curious - is there a way to avoid this file altogether?

Reply:
I removed the file and added it to .gitignore